src/sql-parser/src/parser.rs

```diff
@@ -4650,7 +4650,22 @@ impl<'a> Parser<'a> {
         } else {
             None
         };
-        Ok(OrderByExpr { expr, asc })
+
+        let nulls_last = if self.parse_keyword(NULLS) {
+            Some(match self.expect_one_of_keywords(&[FIRST, LAST])? {
+                FIRST => false,
+                LAST => true,
+                _ => unreachable!(),
+            })
```

Comment:
Nit: This could probably be simplified to:
```rust
let last = self.expect_one_of_keywords(&[FIRST, LAST])? == LAST;
Some(last)
```

Reply:
done